canonical/lxd-ui
# Add reasons for disabling delete actions WD-4614

## Done

- added reasons for disabling delete profile, instance, cluster group buttons

Fixes WD-4614

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described in the [Readme](https://github.com/canonical/lxd-ui#setting-up-for-development).
2. Perform the following QA steps:
    - check the hover text of delete buttons for profiles, instances, cluster groups and projects
